### DISCUSSION TITLE
XamlCheckProcessRequirements()  fails because of missing Microsoft.ui.xaml.dll in newly created Uno Net7.0 Project

### DISCUSSION THREAD

**Discussion by ToddGlodek** [unoplatform/uno#12155] [General]
I just upgraded to Visual Studio Version 17.5.5. I feel like I was able to do this before the upgrade - but since the upgrade I see this runtime error on a brand new project created from the template. 

This is what I did :

First run the Update templates - "dotnet new install Uno.Templates" and observe "Success: Uno.Templates::4.8.11 installed the following templates:"

Create a new project :
<img width="540" alt="newProject" src="https://user-images.githubusercontent.com/19764509/236017929-3a3c10d8-79fc-4834-956a-2520eda13369.png">

Do not modify the project in any way.

Then, run the project in "(Unpackaged)" mode and observe that the following error occurs in the generated code:
<img width="658" alt="MissingDLL" src="https://user-images.githubusercontent.com/19764509/236018832-50a5e77a-7750-4905-abd0-9336d0d4d352.png">

NOTE: If you try to run this as "(Package)" - it will not even build.  I have documented that separately as #12154

**Comment by nickrandolph** [comment]
This is a known issue/scenario for the WindowsAppSdk. To run unpackaged you also need to set the following in the Windows csproj
`<WindowsPackageType>None</WindowsPackageType>`
It would appear in testing that there is an issue building when WindowsPackageType is set to None with the latest winappsdk (1.3.230331000) but should work with version 1.2.221109.1 - we'll raise a ticket with MS for this

**Reply by nickrandolph** [reply]
The build error is already logged here: https://github.com/microsoft/WindowsAppSDK/issues/3591
I will close this discussion as using previous version of sdk and adding WindowsPackageType to None should work
